src/routes/+layout.svelte

```diff
@@ -22,10 +24,15 @@
   });
 
   onMount(() => walletStore.initialize());
+
+  onMount(() => scroll.attach());
```

Comment:
Don't know if this is the "Svelte way", but why have 2 onMount implementations?

Reply:
Good question, not necessarily aware of this being "the svelte way" though I've seen it in a few projects.

My thinking was to separate them as distinct mount calls given they're separate concerns, and sequence doesn't matter, which would be a tad less explicit in one mount function. 🤷‍♂️

I can also see why it might look awkward though, if you think it'd be better I'm happy to change it 🙂

Reply:
No no, it's okay like this. I just wanted to know if there was a convention I didn't know about.
Let's do it this way (and just keep the `onMount` implementations "close enough" in the source code so that we can quickly locate them if they are more than one.)